src/routes/safe/components/Apps/index.tsx

```diff
@@ -1,12 +1,30 @@
-import { useSafeAppUrl } from 'src/logic/hooks/useSafeAppUrl'
+import { useHistory } from 'react-router-dom'
+import { useSelector } from 'react-redux'
 
+import { useSafeAppUrl } from 'src/logic/hooks/useSafeAppUrl'
+import { generateSafeRoute, SAFE_ROUTES } from 'src/routes/routes'
+import { getCurrentShortChainName } from 'src/config'
+import { currentSafe } from 'src/logic/safe/store/selectors'
 import AppFrame from './components/AppFrame'
 import AppsList from './components/AppsList'
+import LegalDisclaimer from './components/LegalDisclaimer'
```

Comment:
We normally import absolute to `src/`. You can let VSCode do it [automatically](https://stackoverflow.com/a/53137571) for you.

Reply:
aah fantastic. done